src/config/backpack/operations/update.php

```diff
@@ -37,13 +37,15 @@
 
 /**
  * Before saving the entry, how would you like the request to be stripped?
- * - false - fall back to Backpack's default (ONLY save inputs that have fields)
- * - closure - process your own request (example removes all inputs that begin with underscode).
+ * - false - use Backpack's default (ONLY save inputs that have fields)
+ * - closure - custom request processing (quick, but will stop config caching)
+ * - invokable class - eg. App\Http\Requests\StripBackpackRequest::class.
  *
  * @param  \Illuminate\Http\Request  $request
  * @return array
  */
+    // 'strippedRequest' => App\Http\Requests\StripBackpackRequest::class,
     // 'strippedRequest' => (function ($request) {
     //     return $request->except('_token', '_method', '_http_referrer', '_current_tab', '_save_action');
-    // }),
+    //  }),
```

Comment:
What?! WHAT?! But... why? 

![hmmm_no_idea](https://user-images.githubusercontent.com/1032474/177788483-6eb2e726-1ee1-4486-aa93-2c3eedf2f504.gif)


1) Why does that work? 😅 please ELI5 😅
2) If it does work, I say we should _default_ to running that, instead of the closure. 
3) Does this still get cached? I would suspect not... but then again... we don't need this one cached, so that should be ok. The invokable class doesn't get cached either (just its name), so it's the same thing in that respect.
4) If this work... do we really need the invokable?! I don't think so.

Reply:
I have no idea why this even works... tbh
I would say the invokable seems a bit less hacky than this one 😆

Reply:
I guess this works because it auto-executes the closure (runs the function) and `strippedRequest` get's an array value that can be serialized. Not sure it will work (it can by-pass the `config:cache`) but does it WORK ?

Reply:
Ah right. So basically it gets run (and calculated) one time, then the array is cached. So the `strippedRequest` would forever be the `request` from the `php artisan config:cache` call. _No bueno._ Good catch @pxpm . Nice try @maurohmartinez 😅

Yup, let's go with invokable then 👍